platform/app/services/syncs/create_service.rb

```
@@ -0,0 +1,44 @@
+      schema.each_key do |stream|
+        Sync.create!(
+          connection: @connection,
+          stream_name: stream,
+          status: :queued,
+          sync_mode: determine_sync_mode(schema[stream]),
+          schedule_type: :manual,
+          sync_frequency: DEFAULT_SYNC_FREQUENCY,
+          supported_sync_modes: schema[stream]["supported_sync_modes"]
+        )
+      end
+    end
```

rubocop

[convention] Method has too many lines. [11/10]

(Metrics/MethodLength)